core/src/main/scala/scalaz/IList.scala

```diff
@@ -694,12 +694,12 @@ sealed abstract class IListInstances extends IListInstance0 {
         @tailrec def commaSep(rest: IList[A], acc: Cord): Cord =
           rest match {
             case INil() => acc
-            case ICons(x, xs) => commaSep(xs, (acc :+ ",") ++ A.show(x))
+            case ICons(x, xs) => commaSep(xs, acc ++ "," ++ A.show(x))
```

Comment:
maybe having `Cord.Comma` or (private) `IList.Comma`?

Reply:
I thought about that but all the structures seem to do their own thing with this so I'm not sure.